go/internal/i18n/locate_lang.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
no `main` package in `go/internal`

Reply:
Yes sorry, I was debbuging (it's not ready yet).